__tests__/app.test.js

```diff
@@ -75,6 +75,35 @@ describe("app ", () => {
           });
         });
     });
+    describe("queries", () => {
+      test('respond with a status of 200, and display an array of review object filtered by category "dexterity"', () => {
+        return request(app)
+          .get("/api/reviews?category=dexterity")
+          .expect(200)
+          .then((response) => {
+            expect(Array.isArray(response.body)).toBe(true);
+            expect(response.body[0]).toHaveProperty("category", "dexterity");
```

Comment:
ideally, this test will check the array length, then iterate over the array, checking that **all** elements have the category value `"dexterity"`